### PR TITLE
Refactor auth module

### PR DESCRIPTION
**Pull request by ianrodrigues**
This PR is related to the issue (#376).

Is really there a need to have more than one `UserIsAlreadyActivated` class? And do same for events? 🤔

### CODE REVIEW

**Comment by sebastiandedeyne** [comment]
Thanks! I'll review this Monday :)

**Comment by ianrodrigues** [comment]
ok, any time! :)

File: tests/Browser/Back/PasswordResetTest.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace Tests\Browser\Back;
 
-use App\Services\Auth\Back\User;
+use App\Models\Administrtor;
```
Comment (sebastiandedeyne) [inline tests/Browser/Back/PasswordResetTest.php:5]: Looks like there's a typo in this file :)

File: tests/Browser/Back/PasswordResetTest.php [hunk 2]
```diff
@@ -26,7 +26,7 @@ public function it_can_reset_passwords()
     {
         $newPassword = 'newpassword';
 
-        $user = User::create([
+        $user = Administrtor::create([
```
Comment (sebastiandedeyne) [inline tests/Browser/Back/PasswordResetTest.php:29]: See above

**Review by sebastiandedeyne — CHANGES_REQUESTED** [review]
Awesome! Commented on a typo somewhere.

Before we wrap this up, could you resolve the conflicts, and rename the tables too (`users` and `admnistrators`)?

**Comment by ianrodrigues** [comment]
Yeah, I can do it next Monday or Tuesday.

**Pull request closed without merging**